doc/sphinx/source/quickstart/installation.rst

```diff
@@ -25,7 +25,29 @@ In order to install the Conda package, you will need both conda and Julia
 pre-installed, this is because Julia cannot be installed from conda.
 For a minimal conda installation go to https://conda.io/miniconda.html.
 Installation instructions for Julia can be found on the
-`Julia download page <https://julialang.org/downloads/>`_.
+`Julia download page <https://julialang.org/downloads/>`_. To install and link Julia
+follow these steps:
+
+* First get and extract Julia (here we get Julia 1.0.3):
+
+.. code-block:: bash
+
+    wget https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.3-linux-x86_64.tar.gz
+    tar xfz julia-*-linux-x86_64.tar.gz
+
+* Then link the executable to your miniconda ``bin`` where all other executables are:
+
+.. code-block:: bash
+
+    ln -s $(pwd)/julia-*/bin/julia $(home)/miniconda3/bin/julia
```

Comment:
This holds for the base environment, shouldn't this be: `miniconda3/envs/esmvaltool/bin/` or another env?

Reply:
There is no need to try to link julia to the conda environment: so long as the `julia` executable can be found in one of the directories listed in `PATH` things should work fine

Reply:
I think it would be better to just point users to the site with the julia installation instructions: https://julialang.org/downloads/platform/ or https://julialang.org/downloads/platform/#linux_and_freebsd if we want to be specific. If these instructions change, ours will become outdated.

Reply:
no, if those change we change the documentation too; man, a lot of the users prefer copy-paste instructions and going to see another set of instructions is already tedious for them :beer:

Reply:
> There is no need to try to link julia to the conda environment: so long as the `julia` executable can be found in one of the directories listed in `PATH` things should work fine

correct! change that line if you think it's better to append to $PATH, man, up to you